pkg/networkserver/networkserver_flow_test.go

```diff
@@ -60,7 +60,18 @@ func frequencyPlanMACCommands(macVersion ttnpb.MACVersion, phyVersion ttnpb.PHYV
 				NbTrans:       1,
 			},
 		}
-		if !otaa || phyVersion.Compare(ttnpb.PHY_V1_0_3_REV_A) < 0 {
+		beforeRP001_V1_0_3_REV_A := func(v ttnpb.PHYVersion) bool {
```

Comment:
I've removed the ordering of versions, as `RP002` has clashing definitions for `1.0.0` and `1.0.1`.